modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminDisplayContext.java

```diff
@@ -382,6 +393,14 @@ public long getUserId() {
 		return _userId;
 	}
 
+	public boolean isSwapConfigurationDisabled() {
```

Comment:
Is very hard to read negative boolean condition for humans, better rename it as isSwapConfigurationEnabled

Reply:
I could not agree more! thx